### ISSUE TITLE
Currently broken

### ISSUE THREAD

**Issue by JustArion** [scotow/zippyst#1]
Some of the regexes might have been broken, i'm getting a ScriptContentNotMatching error.

```rs
[arion@kompooter zippyst]$ ls
Cargo.lock  Cargo.toml  README.md  src  target
[arion@kompooter zippyst]$ cargo run 'https://www3.zippyshare.com/v/CDCi2wVT/file.html'
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/main 'https://www3.zippyshare.com/v/CDCi2wVT/file.html'`
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ScriptContentNotMatching', src/bin/main.rs:11:22
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'task has failed', /home/arion/.cargo/registry/src/github.com-1ecc6299db9ec823/smol-0.1.4/src/task.rs:263:55
[arion@kompooter zippyst]$
```

The test link used is that of the examples in https://github.com/scotow/zippyst#examples

**Comment by scotow** [comment]
Hello,

I just released version [1.1.0](https://github.com/scotow/zippyst/releases/tag/1.1.0) that matches their new script.

**Comment by JustArion** [comment]
Epic! Can confirm it works thanks!
